src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -66,7 +66,6 @@ public partial class View : Container, IResourcesProvider
         private Position internalPosition = null;
         private Position2D internalPosition2D = null;
         private Vector3 internalScale = null;
-        private Size internalSize = null;
         private Size2D internalSize2D = null;
```

Comment:
@dongsug-song 
private member variables beginning with 'internal '  in View could be removed.
it would reduce memory a little.
what do you think?

Reply:
yes. that's good idea. 
but could you add some verifying test code or test app?
after testing the verifying code, then we can merge this.
thank you.

Reply:
`internalSize` is used to reduce temporary instance so if remove it, increased temporary memory. 
please profile with .net memory profiler

Reply:
ok, I will do profile after removed.

Reply:
after removing internalXXX,  memory is increased a little.
so I will close this PR.